packages/lib/hooks/useLastUsed.tsx

```diff
@@ -1,15 +1,20 @@
 import { useState, useEffect } from "react";
 
+import classNames from "@calcom/lib/classNames";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { localStorage } from "@calcom/lib/webstorage";
 
-type LoginType = "saml" | "google" | "credentials" | undefined;
+type LoginType = "saml" | "google" | "credentials";
 
 export function useLastUsed() {
-  const [lastUsed, setLastUsed] = useState<LoginType>(() => {
+  const [lastUsed, setLastUsed] = useState<LoginType>();
+
+  useEffect(() => {
     const storedValue = localStorage.getItem("last_cal_login");
-    return storedValue ? (storedValue as LoginType) : undefined;
```

Comment:
@anikdhabal This can cause a hydration error due to HTML mismatch because it will be `undefined` when rendered in the server whereas it can be some value when rendered in the client